src/main/java/com/jcabi/github/RtRepo.java

```diff
@@ -46,15 +50,15 @@
  * @since 0.1
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle ClassFanOutComplexity (10 lines)
- * @todo #923 Let's implement RtRepo.languages().
- *  Now it returns empty iterable but should return list of repo languages.
- *  Uncomment test RtRepoTest.iteratesLanguages() when done.
- *  See https://developer.github.com/v3/repos/#list-languages for API details.
  */
 @Immutable
 @Loggable(Loggable.DEBUG)
 @EqualsAndHashCode(of = { "ghub", "entry", "coords" })
-@SuppressWarnings({ "PMD.TooManyMethods", "PMD.CouplingBetweenObjects" })
+@SuppressWarnings({
+        "PMD.TooManyMethods",
+        "PMD.CouplingBetweenObjects",
+        "PMD.AvoidInstantiatingObjectsInLoops"
```

Comment:
It is better to have suppress closer to the source, here `PMD.AvoidInstantiatingObjectsInLoops` should be for `languages()` method